src/style.css

```diff
@@ -0,0 +1,19 @@
+* {
```

Comment:
`*` selector is used to add this styles to `EACH` element of the file, you don't usually need this and this is too heavy for the big page, that's why this is better to add this style to those elements you really need